tests/test_buffers.py

```diff
@@ -82,15 +82,16 @@ def test_from_python():
         for j in range(m4.cols()):
             assert m3[i, j] == m4[i, j]
 
-    cstats = ConstructorStats.get(m.Matrix)
-    assert cstats.alive() == 1
-    del m3, m4
-    assert cstats.alive() == 0
-    assert cstats.values() == ["2x3 matrix"]
-    assert cstats.copy_constructions == 0
-    # assert cstats.move_constructions >= 0  # Don't invoke any
-    assert cstats.copy_assignments == 0
-    assert cstats.move_assignments == 0
+    if not env.GRAALPY:
```

Comment:
What do you think about using:

if env.GRAALPY:
    pytest.skip("ConstructorStats is incompatible with GraalPy.")

here (and in similar places elsehwere)? — Then we don't have to indent the entire block below.

Reply:
That will report that it skipped the whole test, when half of it did run. If you do that, then you could just put it as a decorator and actually skip the whole test. Maybe it's possible to split this in half?

Reply:
I just tried out my suggestion locally. The pytest output looks fine to me (ignoring the existing print noise):

```
test_buffers.py ................ss.........### test_submodule_buffers(module_&)::Matrix @ 0x3175a00 destroyed 2x3 matrix
### test_submodule_buffers(module_&)::Matrix @ 0x3175b80 destroyed 5x4 matrix
                                                                                                [100%]

============================================================ short test summary info =============================================================
SKIPPED [1] test_buffers.py:86: ConstructorStats is incompatible with GraalPy.
SKIPPED [1] test_buffers.py:125: ConstructorStats is incompatible with GraalPy.
========================================================= 25 passed, 2 skipped in 0.02s ==========================================================
```

> That will report that it skipped the whole test

What makes you think that?

The main motivation for my suggestion was that the diff will be less noisy:

```diff
diff --git a/tests/test_buffers.py b/tests/test_buffers.py
index 535f33c2..8f0c849e 100644
--- a/tests/test_buffers.py
+++ b/tests/test_buffers.py
@@ -82,6 +82,9 @@ def test_from_python():
         for j in range(m4.cols()):
             assert m3[i, j] == m4[i, j]
 
+    if env.GRAALPY:
+        pytest.skip("ConstructorStats is incompatible with GraalPy.")
+
     cstats = ConstructorStats.get(m.Matrix)
     assert cstats.alive() == 1
     del m3, m4
@@ -118,6 +121,9 @@ def test_to_python():
     mat2[2, 3] = 5
     assert mat2[2, 3] == 5
 
+    if env.GRAALPY:
+        pytest.skip("ConstructorStats is incompatible with GraalPy.")
+
     cstats = ConstructorStats.get(m.Matrix)
     assert cstats.alive() == 1
     del mat
```

> Maybe it's possible to split this in half?

I recommend against doing that in this PR.